### PR TITLE
Wrap docs

### PR DESCRIPTION
**Pull request by pileks**
- Allow builds to happen without Node ENV variables present
- Add AA Wrap README. Pls read

### CODE REVIEW

File: wraps/account-abstraction/docs/readme.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Safe AA Wrap
+
+The Safe AA Wrap is an implementation of the [Safe{Core} Account Abstraction SDK](https://docs.safe.global/learn/safe-core/safe-core-account-abstraction-sdk) in the form of a Polywrap wrap.
+
+As with any wrap, you can use the Safe AA Wrap within any environment where you can use the [Polywrap Client](https://docs.polywrap.io/clients).
+
+## Configuration
+
+The AA Wrap requires you to set your wallet's private key and the Ethereum RPC URL in order to be able to deploy your Safe contract and call its functions.
```
Comment (cbrzn) [inline wraps/account-abstraction/docs/readme.md]:
```suggestion
The AA Wrap allows you to pass a signer (through browser or with private key) in order to be able to deploy your Safe contract and call its functions.
```

File: wraps/relay/docs/readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+## Safe Relay Wrap
```
Comment (cbrzn) [inline wraps/relay/docs/readme.md:1]: i guess this is todo?

File: wraps/account-abstraction/docs/readme.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+      "14"
+    ]
+  }
+});
+```
+
+The Account Abstraction Wrapper's `relayTransaction` method allows you to make a gasless transaction by performing the following steps:
+1. It checks whether there's a deployed Safe contract on your predicted address.
+2. It deploys a Safe contract to the predicted address if there isn't one.
+3. It executes the gasless (sponsored) transaction using the Gelato Relay.
```
Comment (cbrzn) [inline wraps/account-abstraction/docs/readme.md]: actually it allows to do sponsored transactions or user can pay it with erc20 https://docs.gelato.network/developer-services/relay/non-erc-2771/callwithsyncfee

File: wraps/account-abstraction/docs/readme.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+      data: encodeResult.data, // Invocation data.
+      operation: "0"
+    },
+    options: {
+      gasLimit: "300000",
+      isSponsored: true
+    },
+    config: {
+      // Replace this with your own salt nonce. This is used to deploy your safe contract if there is none.
+      saltNonce: "0x258802387238728372837283771"
```
Comment (cbrzn) [inline wraps/account-abstraction/docs/readme.md]: nit: salt nonce does not need to be defined, if it is not sent, the wrapper will take `Date.now()` as salt (and as hex)

File: wraps/relay/docs/readme.md (hunk 2 above)
Comment (pileks) [reply wraps/relay/docs/readme.md:1]: Yes!

Comment (pileks) [reply wraps/relay/docs/readme.md:1]: Exacto

File: wraps/account-abstraction/docs/readme.md (hunk 4 above)
Comment (pileks) [reply wraps/account-abstraction/docs/readme.md]: Added to comments

**Review by kev-ngo — COMMENTED** [review]
@pileks 

* Can we add a note: > ⚠️ Note:  You should be using a `RPC_URL` and `PRIVATE_KEY` meant for TESTING purposes only!

* Which file should this configuration code go in?

**Pull request merged**